src/components/GordonDialogBox/index.tsx

```diff
@@ -69,6 +69,11 @@ const GordonDialogBox = ({
             {cancelButtonName ?? 'Cancel'}
           </Button>
         )}
+        {closeButtonClicked && (
+          <Button variant="contained" color="neutral" onClick={closeButtonClicked}>
+            {closeButtonName ?? 'Close'}
+          </Button>
+        )}
```

Comment:
I'm having trouble imagining a scenario where a dialog should have both a Cancel button and a Close button. Is there some benefit I'm not thinking of in allowing both of these to exist?

Reply:
Looking through the rest of 360 it seems that almost every instance of GordonDialogBox was a form that requested user input, so the Cancel button was there along with a Submit button just in case the user changed their mind. The mailbox instructions is not a form, however, so this would probably be the only place right now where the user would use a Close button instead of the Cancel button.

Reply:
^agreed with above, I didn't realize there was a cancel button.